### DISCUSSION TITLE
Inconsistency with Tire Stints

### DISCUSSION THREAD

**Discussion by adityakotha03** [br-g/openf1#61] [Q&A]
The issue currently is that all the tire data in 2024 for the final lap are off by one value. For instance, the 'lap_end' for the last entry for all drivers has an extra lap added, but this was only for the data in 2024 and not in 2023. Also, the lap data is empty, but I believe this was already mentioned in previous discussions. Another inconsistency is that after every race, we are having duplicate values for race meeting keys; for example, the Chinese Grand Prix is listed twice in the meetings. This also happened during the Japanese Grand Prix but was fixed later.

**Comment by br-g** [comment]
Hello,

The last lap of each race is the cool-down lap I believe (in 2024 at least). That's why its first sector is very slow, and it has missing information.
I don't know why there are differences between 2023 and 2024!

The duplicate meetings was a bug. It has been fixed now.
